PHP/PHP Source.sublime-syntax

```diff
@@ -1349,7 +1349,7 @@ contexts:
       scope: support.function.ctype.php
     - match: (?i)\bcurl_(setopt(_array)?|c(opy_handle|lose)|init|e(rr(no|or)|xec)|version|getinfo|multi_(select|close|in(it|fo_read)|exec|add_handle|remove_handle|getcontent))\b
       scope: support.function.curl.php
-    - match: (?i)\b(str(totime|ptime|ftime)|checkdate|time(zone_(name_(from_abbr|get)|transitions_get|identifiers_list|o(pen|ffset_get)|version_get|location_get|abbreviations_list))?|idate|date(_(su(n(set|_info|rise)|b)|create(_from_format)?|time(stamp_(set|get)|zone_(set|get)|_set)|i(sodate_set|nterval_(create_from_date_string|format))|offset_get|d(iff|efault_timezone_(set|get)|ate_set)|parse(_from_format)?|format|add|get_last_errors|modify))?|localtime|g(et(timeofday|date)|m(strftime|date|mktime))|m(icrotime|ktime))\b
+    - match: (?i)\b(str(totime|ptime|ftime)|checkdate|(hr)?time(zone_(name_(from_abbr|get)|transitions_get|identifiers_list|o(pen|ffset_get)|version_get|location_get|abbreviations_list))?|idate|date(_(su(n(set|_info|rise)|b)|create(_from_format)?|time(stamp_(set|get)|zone_(set|get)|_set)|i(sodate_set|nterval_(create_from_date_string|format))|offset_get|d(iff|efault_timezone_(set|get)|ate_set)|parse(_from_format)?|format|add|get_last_errors|modify))?|localtime|g(et(timeofday|date)|m(strftime|date|mktime))|m(icrotime|ktime))\b
```

Comment:
These condensed regexes are unfortunately rather difficult to read and write. We should really expand, them, but that is not necessary for this PR.

This change results in all of the various `time*()` functions having an `hrtime*()` variant. For example `hrtimezone_ abbreviations_list()` is marked as a support function, whereas it doesn't actually exist.

Reply:
I've updated this now :)

@jfcherng I'll run your script and submit a new PR once this one is approved.